### PR TITLE
Fir 26

### PR DESCRIPTION
**Pull request by AnriDumbadze**

### CODE REVIEW

File: app/Components/Aside/Aside.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import styles from './Aside.module.scss'
+import Icon from '../Icon/Icon';
+import MenuItem from '../MenuItem/MenuItem';
+
+
+const AsideMenu = () => {
+    return (
+
```
Comment (lursmana1) [inline app/Components/Aside/Aside.tsx]: fix spacing with prettier

File: app/Components/NavItem/NavItem.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      return styles.sale;
+    }
+  };
+
+  return (
+    <div className={saleStylesRender()}>
+    {props.visible && (
+      <div className={styles.nav}>
+      <div className={styles.navContent}>
+        <div className={styles.navItem} style={ItemStyle}>
```
Comment (lursmana1) [inline app/Components/NavItem/NavItem.tsx]: u can directly pass the background color directly without external constant

File: app/Components/Navigation/Nav.module.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.nav {
```
Comment (lursmana1) [inline app/Components/Navigation/Nav.module.scss:1]: i dont think these styles are enough

File: app/globals.css [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
+  color: #333;
+}
+
+h1, h2, h3, h4, h5, h6 {
```
Comment (lursmana1) [inline app/globals.css]: why u need external commands for this u have * over it

File: app/page.tsx [hunk 5]
```diff
@@ -1,13 +1,21 @@
...
-import Button from './Components/Button/Button';
+import AsideMenu from './Components/Aside/Aside';
+import styles from './page.module.scss'
+import Navigation from './Components/Navigation/Nav';
 
 const Home = () => {
     return (
-      <>
+      
+      <div className={styles.mainContent}>
```
Comment (lursmana1) [inline app/page.tsx]: Nice

File: app/Components/NavItem/NavItem.tsx (hunk 2 above)
Comment (AnriDumbadze) [reply app/Components/NavItem/NavItem.tsx]: but the backgrand is different in pages so I made it with prop

Comment (lursmana1) [reply app/Components/NavItem/NavItem.tsx]: no ItemStyle itself is external u can pass it like style ={backgroundColor : props.background}

File: app/Components/Navigation/Nav.module.scss (hunk 3 above)
Comment (AnriDumbadze) [reply app/Components/Navigation/Nav.module.scss:1]: It worked in the design with different things i tested what do i add

**Pull request merged**